include/aktualizr-lite/tuf/tuf.h

```diff
@@ -0,0 +1,293 @@
+#ifndef AKLITE_TUF_TUF_H_
+#define AKLITE_TUF_TUF_H_
+
+#include <iostream>
+#include <string>
+
+#include <boost/algorithm/string.hpp>
+#include <boost/filesystem.hpp>
+#include <boost/program_options.hpp>
+#include <boost/program_options/variables_map.hpp>
+#include <boost/property_tree/ptree.hpp>
+
+#include "crypto/keymanager.h"
+#include "http/httpclient.h"
+#include "json/json.h"
+#include "libaktualizr/config.h"
+#include "storage/invstorage.h"
+#include "uptane/fetcher.h"
+#include "uptane/imagerepository.h"
+
+#include "aktualizr-lite/tuf/tuf.h"
+
+namespace aklite {
+namespace tuf {
+
+// TODO: consider switching to a single fetchRole
```

Comment:
Do you still plan on adding a single `fetchRole` method?

Reply:
@detsch What's your plan on this?

Reply:
Let's keep the current approach, with multiple methods.